Gemfile

```diff
@@ -38,6 +38,9 @@ gem 'omniauth'
 gem 'request_store', "~> 1.1.0"
 gem 'gravatar_image_tag', '~> 1.2.0'
 
+gem 'warden', '~> 1.2'
```

Comment:
didn't we decide https://github.com/opf/openproject/pull/2104#issuecomment-99890764? i think `warden` is pretty stable.